content/remote-development/release-notes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Release Notes v0.1.0
+menu:
+  main:
+    parent: 'remote-development'
+    weight: 50
+
+weight: 50
```

Comment:
Didn't realise `weight` was for ordering, they could have called it `order-item` 😕  I'll change it 😄